examples/cephfs/secret.yaml

```diff
@@ -4,11 +4,11 @@ kind: Secret
 metadata:
   name: csi-cephfs-secret
   namespace: default
-data:
+stringData:
   # Required for statically provisioned volumes
-  userID: BASE64-ENCODED-VALUE
-  userKey: BASE64-ENCODED-VALUE
+  userID: <plaintext ID>
+  userKey: <Ceph auth key corresponding to ID above>
```

Comment:
why are we moving away from BASE64 to plain text here?

Reply:
See usability discussion here, https://github.com/ceph/ceph-csi/pull/395#discussion_r289988124

Basically, moving away to make it easy for an user t ospecify plain text IDs and key is already base64 encoded by Ceph.